docs/users/overview.adoc

```diff
@@ -10,10 +10,11 @@ You will quickly have an overview of the `bpmn-visualization` capabilities.
 === Usage
 
 `bpmn-visualization` can be directly used in HTML pages or integrated in Javascript and TypeScript applications. It is distributed as a https://npmjs.org/package/bpmn-visualization[npm package]
-in various bundle formats.
+in various bundle formats (IIFE, ESM & CommonJS).
```

Comment:
⚠️ not detailing the formats was not an oversight: this is already documented in the README and here, we didn't want to duplicate the information.